app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -380,6 +384,32 @@ def handle_successful_status_update(enrollment, response)
     )
   end
 
+  def handle_call_in_needed(enrollment, response)
+    proofed_at = parse_usps_timestamp(response['transactionEndDateTime'])
+    enrollment_outcomes[:enrollments_passed] += 1
+    analytics(user: enrollment.user).idv_in_person_usps_proofing_results_job_enrollment_updated(
+      **enrollment_analytics_attributes(enrollment, complete: true),
+      **response_analytics_attributes(response),
+      passed: true,
+      reason: 'Successful status update',
```

Comment:
the reason we assign to passed enrollments is `successful status update`, since these enrollments have passed should we use the same reason or should we update the reason to something like `successful status update with fraud review pending`?

Reply:
I decided to update the reason to be more reflective of the fact that is was in review when passed